README.md

```diff
@@ -13,19 +10,9 @@ compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=fl
 Add to your Podfile:
 
 ```ruby
-pod 'Alamofire-Gloss'
+pod 'Alamofire-Gloss', :git => 'https://github.com/jpunz/Alamofire-Gloss.git'
```

Comment:
going back to changing the podspec––i'll handle updating the project meta-data/readme/etc.. before i update the cocoapod version.